platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/internal/EventImpl.java

```diff
@@ -239,55 +233,24 @@ private void setDefaultValues() {
     }
 
     /**
-     * Set this event's generation to be 1 more than the max of its parents.
+     * Set the consensusTimestamp to an estimate of what it will be when consensus is reached even
+     * if it has already reached consensus. Callers are responsible for checking the consensus
+     * systemIndicesStatus of this event and using the consensus time or estimated time
+     * appropriately.
      *
-     * @deprecated
-     */
-    @Deprecated(forRemoval = true) // there is no need to store the events generation inside consensusData
-    public void updateConsensusDataGeneration() {
-        consensusData.setGeneration(baseEvent.getHashedData().getGeneration());
-    }
-
-    /**
-     * Calls {@link InternalEventData#clear()}
-     */
-    public void clear() {
-        internalEventData.clear();
-    }
-
-    /**
-     * Set the consensusTimestamp to an estimate of what it will be when consensus is reached even if it has already
-     * reached consensus. Callers are responsible for checking the consensus systemIndicesStatus of this event and using
-     * the consensus time or estimated time appropriately.
-     * <p>
-     * Estimated consensus times are predicted only here and in Platform.estimateTime().
+     * <p>Estimated consensus times are predicted only here and in Platform.estimateTime().
      *
-     * @param selfId                    the ID of this platform
-     * @param avgSelfCreatedTimestamp   self event consensus timestamp minus time created
+     * @param selfId the ID of this platform
+     * @param avgSelfCreatedTimestamp self event consensus timestamp minus time created
      * @param avgOtherReceivedTimestamp other event consensus timestamp minus time received
+     * @deprecated this is only used for SwirldState1 which we no longer support, and it did not do
+     *     any estimates
      */
+    @SuppressWarnings("unused")
+    @Deprecated(forRemoval = true)
```

Comment:
We should discuss if this estimated time is needed anymore. It was semi-important for the SwirldState1 case where transactions were applied to the current state in an estimated order. But now, the only pre-consensus handler is one that collects state signature transactions. The consequence of events being processed pre-consensus in a worse order than they would be based on this estimated time is a slight delay of the state signature being added to a state's signature set. It's here now, so maybe we leave it and write a ticket to remove it later when we can do some testing to determine what, if any, impact this currently has.

Reply:
added #8606